v22.1/api-support-policy.md

```diff
@@ -0,0 +1,89 @@
+---
+title: API Support Policy
+summary: Learn about Cockroach Labs' policy for supporting CockroachDB APIs.
```

Comment:
Nit: If you must use a possessive here, I think `Labs's` might actually be correct. Plural possessive is difficult for folks for whom English isn't their primary language.